Sources/Algorithms/Trim.swift

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Algorithms open source project
+//
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+//
+//===----------------------------------------------------------------------===//
+
+extension BidirectionalCollection {
+
+  /// Returns a `SubSequence` formed by discarding all elements at the start and end of the Collection
```

Comment:
Nit: I _think_ this project's style is to wrap to 80 columns.

Reply:
I ran this through swift-format (the project doesn't have a format file, so I assume it's the default settings?), but it doesn't seem to wrap comments. Will fix.